inc/saiacl.h

```diff
@@ -1192,6 +1194,7 @@ typedef enum _sai_acl_entry_attr_t
      *
      * @type sai_acl_field_data_t sai_uint16_t
      * @flags CREATE_AND_SET
+     * @isvlan false
```

Comment:
can we make isvlan=false by default so that we do not need to mark this for all uint16?